it/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: news_post
+title: "Ruby 2.3.0-preview2 Released"
+author: "naruse"
+translator: "simo2409"
+date: 2015-12-11 14:00:00 +0000
+lang: it
+---
+
+Siamo lieti di annunciare il rilascio di Ruby 2.3.0-preview2.
+
+Ruby 2.3.0-preview2 è la seconda anteprima di Ruby 2.3.0.
+Sono state aggiunte molte nuove funzionalità e miglioramenti.
+
+E' stato aggiunto il [Frozen String Literal Pragma](https://bugs.ruby-lang.org/issues/11473).
+Su Ruby 2.1 `"str".freeze` è
+stato ottimizzato per ridurre il numero di oggetti allocati in memoria. Ruby 2.3
+introduce un nuovo commento 'magico' e un'opzione da linea di comando che
+permette di congelare tutte le stringhe nel codice sorgente.
+Inoltre per poter debuggare le proprie applicazioni, potete individuare dove sia
+stato creato l'oggeto che causa l'errore `"can't modify frozen String"` passando
+l'opzione `--debug=frozen-string-literal`.
+
+Il [safe navigation operator](https://bugs.ruby-lang.org/issues/11537)
+([chiamato anche lonely operator](https://instagram.com/p/-M9l6mRPLR/)) `&.`,
+che esiste già in linguaggi come C#, Groovy e Swift, è stato introdotto per
+rendere più semplice la gestione di oggetti che potrebbero essere `nil`,
+scrivendo ad esempio `obj&.foo`. Inoltre sono stati aggiunti `Array#dig` e
+`Hash#dig`.
+
+E' stata anche aggiunta la gemma
+[did_you_mean](https://bugs.ruby-lang.org/issues/11252), che mostra le possibili
+alternative quando si incontrano `NameError` o `NoMethodError` per rendere più
+semplice il debug delle nostre applicazioni.
+
+Sono stati aggiunti [RubyVM::InstructionSequence#to_binary e .load_from_binary](https://bugs.ruby-lang.org/issues/11788)
+come funzionalità sperimentali.
+Con queste funzionalità, possiamo creare un sistema di pre-compilazione ISeq (bytecode).
+
+Ruby 2.3 include molti fix per migliorare le performance.
+Ad esempio,
+[ottimizzazzione Proc#call](https://bugs.ruby-lang.org/issues/11569),
+[reconsidering method entry data structure](https://bugs.ruby-lang.org/issues/11278),
```

Comment:
@simo2409 @stomar we should probably translate all link content here? or maybe none of them but it's proabably better to be more consistent.